draft-mishra-scone-usecase.md

```diff
@@ -67,142 +67,161 @@ informative:
+multiple Customer Premises Equipment (CPE) devices.
+
+Accordingly, Applicability and Manageability considerations must encompass a wide range of access-network
+scenarios, each of which handles per-flow rate limiting differently. This document first presents generic
+considerations for the SCONE protocol and then provides network-specific guidance where throughput
+advisory signaling can enhance both resource utilization and user experience.
 
 # Terms and Definitions
 
-This document uses terms and definitions described in {{I-D.ietf-scone-protocol}}, some more terms and definitions are described below in this section. 
+This document uses terms and definitions described in {{I-D.ietf-scone-protocol}},
+and additionally introduces terms  in this document are described in this section.
 
-- 4G - Fourth Generation mobile network technology, also known as Long-Term Evolution (LTE), defined by the 3rd Generation
+- 4G - Fourth Generation mobile network
+The technology also known as Long-Term Evolution (LTE), defined by the 3rd Generation
 Partnership Project (3GPP).
 
 - 5G - Fifth Generation Mobile Networks
 The fifth generation of cellular mobile network technology defined by 3GPP.
 
-- APN - Access Point Name 
-The Access Point Name (APN) determines the specific Packet Data Network Gateway (PDN-GW in 4G/LTE) that the mobile device
-should use to access a service. The gateway acts as the access point to external networks such as the public internet or a
-private network. Different APNs can be used to provide different services, access privileges, or Quality of Service to a user's
-device.
-
-- Adaptive Bit-Rate (ABR) Video
-Video streaming technology that adjusts video quality dynamically based on network conditions.
-
-- BNG (Broadband Network Gateway)
-A network element that serves as the access point for subscribers in wireline broadband networks. It establishes and manages
-subscriber sessions, aggregates traffic from multiple subscriber access nodes, and routes this traffic to the service
-provider's core network. BNG functions include subscriber authentication, IP address assignment, policy enforcement, and
-quality of service management. It typically supports subscriber session protocols such as DHCP, PPPoE, or IPoE, and interacts
+- APN - Access Point Name
+Is a logical label used in 4G/LTE networks to identify
+the external data network that a User Equipment (UE) requests to connect to. The APN specifies the
+target data network (e.g., the Internet or a private network) and is used by the network core to
+select the appropriate gateway (P-GW) and establish the corresponding data connection.
+It is functionally equivalent to the Data Network Name (DNN) used in 5G systems.
+
+- (ABR) - Adaptive Bit-Rate
+A media delivery technique that dynamically adjusts the quality of content (e.g., video) in real-time based on the
+available network bandwidth.
+
+- Application End-point
```

Comment:
We are defining Application "End-point" but using Application "endpoint". If we are defining "end-point" then we need change where we use "endpoint".